nerfstudio/data/dataparsers/nerfstudio_dataparser.py

```diff
@@ -399,10 +388,45 @@ def _generate_dataparser_outputs(self, split="train"):
 
             if ply_file_path:
                 sparse_points = self._load_3D_points(ply_file_path, transform_matrix, scale_factor)
```

Comment:
You need to annotate the output type of function `_load_3D_points", otherwise the pyright would complains.

Reply:
done